bitnami/kaniko/1/debian-12/Dockerfile

```
@@ -55,12 +58,17 @@ COPY rootfs /
 COPY --from=builder /opt/bitnami/kaniko/.spdx-kaniko.spdx /opt/bitnami/kaniko/.spdx-kaniko.spdx
 COPY --from=builder /opt/bitnami/kaniko/licenses /opt/bitnami/kaniko/licenses
 COPY --from=builder /out /
+COPY --from=busybox /bin /busybox
```

You do not need the busybox image, you can copy the folder from the builder one

```suggestion
COPY --from=builder /bin /busybox
```